docs/docs/adrs/adr-018-permissionless-ics.md

```diff
@@ -0,0 +1,270 @@
+---
+sidebar_position: 19
+title: Permissionless ICS
+---
+# ADR 18: Permissionless Interchain Security
+
+## Changelog
+* 27th of June, 2024: Initial draft
+
+## Status
+
+Proposed
+
+## Context
+Currently, a consumer chain can join _Interchain Security_ (ICS) only through a [governance proposal](../features/proposals.md).
+A governance proposal was needed before the introduction of [Partial Set Security](../features/partial-set-security.md) (PSS)
+because validators were required to validate a consumer chain. However, after the introduction of PSS, a consumer chain can
+be either _Top N_ or _Opt In_. If a chain is an Opt In chain, then no validator is required to validate this chain unless they choose to.
+Because of this, we can launch an Opt In consumer chain without going through a governance proposal.
+
+This ADR presents _Permissionless_ ICS, a way in which an [_Opt In_](adr-015-partial-set-security.md) consumer chain can join
+ICS without needing a governance proposal but by simply issuing a transaction.
+
+## Decision
+
+### The Phases of a Consumer Chain
+In Permissionless ICS, launching an Opt In chain is **only** possible through a transaction and not through a `ConsumerAdditionProposal`.
```

Comment:
That's a good question. My view is that everything _Opt In_-chain related should be done through transactions and everything _Top N_-chain related should be done through governance proposals. This creates a nice distinction between those two.

I do not see a reason why the community would want to remove an Opt In chain because noone is forced to validate it. In case a consumer chain attempts to act maliciously, or so, we probably should have a coordinated upgrade to kick it out as soon as possible and not through a gov proposal